test/models/user_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```

Comment:
Test cases should be in rspec

Reply:
but we havent add rspec in our project till now